contracts/Builders/SimpleBuilder.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/ISimpleProvider.sol";
+import "../interfaces/ILockDealNFT.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+
+/// @title SimpleBuilder contract
+/// @notice This contract is used to create mass lock deals(NFTs)
+contract SimpleBuilder {
+    ILockDealNFT public lockDealNFT;
+    ISimpleProvider dealProvider; // for project owner
+
+    constructor(ILockDealNFT _nft) {
+        lockDealNFT = _nft;
+    }
+
+    struct UserPool {
+        address user;
+        uint256[] params;
```

Comment:
this need to be uint256 - not array.
if we do lock for some TGE - No need to pass the TGE time for the Number of users times.